app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java

```diff
@@ -67,9 +67,8 @@
         if (session.room != null) {
             this.roomName = session.room.name;
         }
-        if (session.lang != null) {
-            this.languageId = session.lang.toUpperCase();
-        }
+       
+        this.languageId = session.lang != null ? session.lang.toUpperCase() : Session.LANG_EN_ID;
```

Comment:
I think it's better to hide language tag and speaker name.
Can you add `languageVisibility` in this class and bind it to `view_session.xml`? 🙇